internal/broker/broker.go

```diff
@@ -418,6 +418,8 @@ func (b *Broker) IsAuthenticated(sessionID, authenticationData string) (string,
 }
 
 func (b *Broker) handleIsAuthenticated(ctx context.Context, session *sessionInfo, authData map[string]string) (access string, data isAuthenticatedDataResponse) {
+	defer decorateErrorMessage(&data, fmt.Sprintf("failed to authenticate user %q", session.username))
+
```

Comment:
What's the reasoning behind decorating the error message with this string? Do you expect it to be more helpful to the user?

Reply:
Exactly. I'm still not sure we should print this or not. I'm fine with just appending "failed to authenticate user" to the error also.

Reply:
No strong opinion, but until we notice in practice that mentioning the username would have helped, I prefer to avoid logging and returning personal identifiers in messages.

Reply:
https://github.com/ubuntu/authd-oidc-brokers/pull/115/commits/c67a78cb0a0c7f1f7879660ceffe141fb7914490 LGTM, feel free to squash that into the other commit